pkg/actors/actors.go

```diff
@@ -620,38 +645,37 @@ func (a *actorsRuntime) TransactionalStateOperation(ctx context.Context, req *Tr
 				Operation: state.Delete,
 			})
 		default:
-			return errors.Errorf("operation type %s not supported", o.Operation)
+			return fmt.Errorf("operation type %s not supported", o.Operation)
 		}
 	}
 
 	policy := a.resiliency.ComponentOutboundPolicy(ctx, a.storeName, resiliency.Statestore)
+	stateReq := &state.TransactionalStateRequest{
+		Operations: operations,
+		Metadata:   metadata,
+	}
 	return policy(func(ctx context.Context) error {
-		return a.transactionalStore.Multi(&state.TransactionalStateRequest{
-			Operations: operations,
-			Metadata:   metadata,
-		})
+		return a.transactionalStore.Multi(stateReq)
 	})
 }
 
 func (a *actorsRuntime) IsActorHosted(ctx context.Context, req *ActorHostedRequest) bool {
 	key := constructCompositeKey(req.ActorType, req.ActorID)
-	exists := false
 	var policy resiliency.Runner
 	if a.isResiliencyEnabled {
 		policy = a.resiliency.BuiltInPolicy(ctx, resiliency.BuiltInActorNotFoundRetries)
 	} else {
 		noOp := resiliency.NoOp{}
 		policy = noOp.BuiltInPolicy(ctx, resiliency.BuiltInActorNotFoundRetries)
 	}
-	policy(func(ctx context.Context) error {
-		_, exists = a.actorsTable.Load(key)
-
+	err := policy(func(ctx context.Context) error {
+		_, exists := a.actorsTable.Load(key)
 		if !exists {
-			return fmt.Errorf("Actor")
+			return errors.New("Actor")
```

Comment:
Should this error message provide more information?

Reply:
I agree but that was there before (I just changed the method name) and I have no idea what  the original author meant this to be

Reply:
Sure, but since it's being touched something like `fmt("actor %s %s not found in actor table", req.ActorType, req.ActorID)` might be handy :-)

Reply:
TBH I don't think that matters. If you look below, we just check to see if `err != nil`. The actual error message is ignored. 
So I just changed it to a message that's much better ;)